generated/rflx-rflx_generic_types.adb

```diff
@@ -1,246 +1,300 @@
 pragma Style_Checks ("N3aAbcdefhiIklnOprStux");
 
-with RFLX.RFLX_Lemmas;
-
-pragma Warnings (Off, """Ada.Numerics.Big_Numbers.Big_Integers"" is an Ada 202x unit");
-
-with Ada.Numerics.Big_Numbers.Big_Integers; use Ada.Numerics.Big_Numbers.Big_Integers;
-
-package body RFLX.RFLX_Generic_Types is
-
-   package Conversions is new Signed_Conversions (Long_Integer);
-
-   function Extract (Data : Bytes;
-                     Ofst : Offset) return Value
+with RFLX.RFLX_Utils; use RFLX.RFLX_Utils;
```

Comment:
I agree that use clauses for packages should be avoided. But what are the arguments for the total ban?

Unfortunately, `U64` cannot be moved into `RFLX`, as the top level package is not always created.

Reply:
> I agree that use clauses for packages should be avoided. But what are the arguments for the total ban?

When using packages there is no easy way to determine the origin of some entity without checking all `use`ed packages (or using some tool). I consider this a big loss in terms of comprehensibility and its especially problematic when reviewing code on GithHub, which has no Ada support (or any cross referencing support for that matter).

Another issue is naming conflicts that may be introduced. Say you use-withed two packages `Foo` and `Bar` and use a type `Baz` from package `Bar`. If someone introduces `Baz` in `Foo`, this may break your code due to the naming conflict.

I don't see a good rule for acceptable application of `use` for packages, hence the total ban.

Reply:
Fixed by 1ac5717.